scene/gui/text_edit.cpp

```diff
@@ -2065,6 +2065,12 @@ void TextEdit::_input_event(const InputEvent& p_input_event) {
 
 					if (k.mod.shift)
 						_pre_shift_selection();
+#ifdef APPLE_STYLE_KEYS
+					else
```

Comment:
It seems so. Maybe the `#endif`should be before `deselect()`? It's also the case in the other `else` below.